### PR TITLE
Added basic configuration for DragonBoard 410c.

### PR DESCRIPTION
**Pull request by viviansh**
Added support in waf to compile for DragonBoard using --board=dragonboard.

### CODE REVIEW

**Comment by OXINARF** [comment]
@viviansh Much better than before, thank you. Since this is your first-time contribution we can fix things for you, but please read our [contribution guidelines](http://ardupilot.org/dev/docs/submitting-patches-back-to-master.html), specially the parts about one commit per library/vehicle/folder and the commit message format.

Can I ask what this is for? Is there going to be a board with sensors based on this DragonBoard? I'm trying to figure out the usefulness to have this now, a board with no sensors.

**Comment by viviansh** [comment]
@OXINARF Thank you, I will make sure to follow the guidelines better next time. This is part of a project that I'm taking at my university. The plan is to add sensors in the future but for now we wanted to make an initial commit to get things started.

**Comment by viviansh** [comment]
@OXINARF @lucasdemarchi Sorry for bothering you but what is our status with this code review?

File: Tools/ardupilotwaf/boards.py [hunk 1]
```diff
@@ -569,3 +569,12 @@ def __init__(self):
         self.romfs_exclude(['oreoled.bin'])
         self.board_rc = True
         self.param_defaults = '../../../Tools/Frame_params/intel-aero-rtf.param'
+
+
+class dragonboard(linux):
```
Comment (OXINARF) [inline Tools/ardupilotwaf/boards.py]: Please keep Linux boards together, it is easier to look at the file that way

**Comment by lucasdemarchi** [comment]
Any progress? Feel free to re-open the PR when there is.

**Pull request closed without merging**